File: js/invertKeysAndValues.js
// Problem# 194
// Write a function that inverts the keys and values of an object.
// https://edabit.com/challenge/YTECpnCCeJsYqYvfF

const invert = (obj) =>
  Object.fromEntries(Object.entries(obj).map(([key, value]) => [value, key]));

module.exports = invert;

File: python/invert_keys_and_values.py
"""
Invert Keys and Values

Write a function that inverts the keys and values of a dictionary.

https://edabit.com/challenge/Axim3Ld5zu9RFLZKr
"""
from typing import Dict


def invert(dct: Dict) -> Dict:
    return {value: key for key, value in dct.items()}
